docs/pages/includes/application-access/azure-tsh.mdx

```diff
@@ -0,0 +1,145 @@
+Now that you have authorized your Teleport user to assume the `teleport-azure`
+identity, you can use Teleport to authenticate to Azure's APIs and execute
+commands against it via the `az` CLI.
+
+### List your Azure CLI application
+
+Verify that your Teleport user can see the `azure-cli` application you
+registered earlier:
+
+```code
+$ tsh apps ls
+Application Description Type Public Address                 Labels
+----------- ----------- ---- ------------------------------ -------------------
+azure-cli               HTTP azure-cli.teleport.example.com teleport.dev/origin
+```
+
+### Log in to use an Azure CLI
+
+Log in to the application, specifying that you would like to assume the
+`teleport-azure` identity:
+
+```code
+$ tsh apps login azure-cli --azure-identity teleport-azure
+```
+
+This command validates the value of the `--azure-identity` flag against the ones
+the user is authorized to assume. The value of the flag can either be the full
+URI of the identity (e.g., the URI you copied earlier in this guide) or the name
+of the identity, e.g., `teleport-azure`. 
+
+A user can omit the `--azure-identity` flag if they are only authorized to
+access a single Azure identity, but otherwise an empty `--azure-identity` will
+result in an error.
```

Comment:
Technically omitting the `--azure-identity` flag will be seen as an empty value by `tsh`, but I'd rephrase this as the flag being "missing" since this is the most likely cause of an empty value.